evcxr_jupyter/src/core.rs

```diff
@@ -36,6 +38,7 @@ pub(crate) struct Server {
     latest_execution_request: Arc<Mutex<Option<JupyterMessage>>>,
     shutdown_requested_receiver: Arc<Mutex<crossbeam_channel::Receiver<()>>>,
     shutdown_requested_sender: Arc<Mutex<crossbeam_channel::Sender<()>>>,
+    command_history: Vec<(&'static str, &'static str)>,
```

Comment:
Do we ever use the commands that were executed besides the most recent one? If we only ever need the most recent one, then do we need to have this in an field, or could it just be a local variable / argument in the code that needs it?